### DISCUSSION TITLE
¿Por qué el menú que posee un usuario se ve colapsado en la ventana al 100% que un 90%?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#73] [2. Questions and Answers] [answered]
No sé si corresponde o está definido de esa forma. Me parece un poco extraño que deba disminuir el aumento hasta el 90% para que pueda ver el nombre del menú / grupos que tenerlo al 100%: ¿Está bien así?. Adjunto las 2 imagenes

**Al tener 100% del navegador**
![image](https://user-images.githubusercontent.com/61419471/114606500-549ce680-9c69-11eb-9ee8-068b7c48fc85.png)

**Al tener 90% del navegador**
![image](https://user-images.githubusercontent.com/61419471/114606531-6088a880-9c69-11eb-9422-fdb69ede014a.png)

Gracias

**Comment by acedward** [comment, marked as answer]
@NicolasPLG depende del ancho de la pantalla en pixeles.
Al ponerlo en 90% alcanzas al siguente nivel de tamaño.

Normalmente se eligen rangos tradicionales, dado el mercado:
![image](https://user-images.githubusercontent.com/3951895/114607207-3aafd380-9c6a-11eb-8ee1-458e55c06056.png)

**Reply by NicolasPLG** [reply]
Entiendo @acedward muchas gracias por la información
